packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -20,13 +21,14 @@ let sdkFolderPath: string;
 let sdkPackagePath: string;
 const execAppOptions = getDefaultExecSyncOptions();
 const packageManager = getPackageManager();
+const describeSkipIfYarn1 = isYarn1() ? describe.skip : describe;
 
-describe('Create new sdk command', () => {
+describeSkipIfYarn1('Create new sdk command', () => {
```

Comment:
can't you do something like:
```suggestion
describe('Create new sdk command', () => {
    test.skip(isYarn1());
```
?

Reply:
I don't think this syntax will work.